test/includes/microcloud.sh

```diff
@@ -533,6 +534,8 @@ reset_system() {
 
     reset_snaps "${name}"
 
+    # Attempt to destroy the zpool as we may have left dangling volumes when we wiped the LXD database earlier.
+    # This is slightly faster than deleting the volumes by hand on each system.
     lxc exec "${name}" -- zpool destroy -f local || true
```

Comment:
So here, with the non-snapshot tests which are primarily optimized for speed, we wipe LXD's database (and entire state directory) out from under it and use the snap to restore it (`snap enable lxd` will restore `/var/snap/lxd/common/` if it doesn't exist). 

This means we won't properly clean up the zpool on each system (there may be dangling volumes) and so we need to manually `zpool destroy -f` to clear everything. We also do this with the `ceph` storage  driver, manually clearing everything out from under `ceph`.

Reply:
> We also do this with the ceph storage driver, manually clearing everything out from under ceph.

Are you referring to how it is done in LXD or some other test suite that does Ceph related cleanups?

Reply:
I was referring to this block. https://github.com/canonical/microcloud/blob/c86c366230bac921eb6f9349295a6f88dfe31a0d/test/includes/microcloud.sh#L596

Reply:
I see, thanks!